graphql2/graphqlapp/alert.go

```diff
@@ -74,9 +74,40 @@ func (a *AlertLogEntry) notificationSentState(ctx context.Context, obj *alertlog
 	case notification.MessageStateSent, notification.MessageStateDelivered:
 		status = "OK"
 	}
+	var details = s.Details
+	switch s.LastStatus {
+	case notification.MessageLastStatusDelivered:
+		if s.Details != "delivered" {
+			details = "delivered:" + s.Details
+		}
+	case notification.MessageLastStatusPending:
+		if s.Details != "pending" {
+			details = "pending:" + s.Details
+		}
+	case notification.MessageLastStatusSending:
+		if s.Details != "sending" {
+			details = "sending:" + s.Details
+		}
+	case notification.MessageLastStatusSent:
+		if s.Details != "sent" {
+			details = "sent:" + s.Details
+		}
+	case notification.MessageLastStatusQueuedRemotely:
+		if s.Details != "queued_remotely" {
+			details = "queued_remotely:" + s.Details
+		}
+	case notification.MessageLastStatusFailed:
+		if s.Details != "failed" {
+			details = "failed:" + s.Details
+		}
+	case notification.MessageLastStatusBundled:
+		if s.Details != "bundled" {
+			details = "bundled:" + s.Details
+		}
+	}
```

Comment:
We can simplify this a bit (and using `State` as in another comment) so that it reads better.


If we structure the code so that the shared/common logic is only written once, it makes the switch easier to scan for just the string/prefix differences.
```
	var prefix string
	switch s.State {
	case notification.MessageStatePending:
		prefix = "pending"
	case notification.MessageStateSending:
		prefix = "sending"
	case notification.MessageStateSent:
		prefix = "sent"
	// ...
	}

	details := s.Details
	if prefix != details {
		details = prefix + ": " + details
	}
```

Also for reference, use `var` for empty declarations and `:=` for assigning declarations.
```
// either
var details string
// or
details := s.Details
```

Using `var foo = ` should only be done (by convention) for package-level variables where `:=` is invalid syntax.